recipes/pulsar/docker-compose.yml

```diff
@@ -54,7 +54,7 @@ services:
     depends_on:
       - pinot-broker
     volumes:
-      - ./plugins/pinot-pulsar-0.12.0-shaded.jar:/opt/pinot/plugins/pinot-stream-ingestion/pinot-pulsar/pinot-pulsar-0.12.0-arm64-shaded.jar
+      - ./plugins/pinot-pulsar-0.12.0-shaded.jar:/opt/pinot/plugins/pinot-stream-ingestion/pinot-pulsar/pinot-pulsar-0.12.0-arm64-shaded.jar 
```

Comment:
@mayankshriv @xiangfu0 is it accurate to update the pinot-pulsar jar to 1.0.0?

Reply:
we can use 1.0.0
put a note that apple is using `arm64`.
Intel` servers are using `amd64`

Reply:
To be sure, you're only referring to this pulsar instance? Noticed when installing pinot on Apple -arm64 tag no longer works only 1.0.0 by itself. Is this your understanding too? Thank you, @xiangfu0!